spec/test-data/translations/glotpress-download/stubs/fr.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Translation-Revision-Date: 2021-03-31 09:54:04+0000
+Plural-Forms: nplurals=2; plural=n > 1;
+Generator: GlotPress/2.4.0-alpha
+Language: fr
+-->
+<resources>
+	<string name="product_attributes_error_saving">Erreur lors de l’enregistrement de vos attributs</string>
+	<string name="shipping_label_payments_cant_edit_warning">Seul le propriétaire du site peut gérer les moyens de paiement des étiquettes d’expédition. Veuillez contacter le propriétaire de la boutique %1$s (%2$s) pour gérer les moyens de paiement.</string>
+	<string name="shipping_label_shipping_method_free_shipping">Livraison gratuite</string>
+	<string name="shipping_label_shipping_method_local_pickup">Point de vente</string>
+	<string name="shipping_label_shipping_method_flat_rate">Forfait</string>
+	<string name="shipping_label_shipping_carrier_flat_fee_banner_message">Le client a payé %1$s (%2$s) pour la livraison</string>
+	<string name="shipping_label_woo_discount_bottomsheet_message" formatted="false">Lors de l’achat d’étiquettes d’expédition avec WooCommerce, vous bénéficiez de 5 à 40 %% de réduction par rapport aux tarifs de la Poste.</string>
+	<string name="shipping_label_woo_discount_bottomsheet_title">Qu’est-ce que la remise WooCommerce  Services ?</string>
+	<string name="shipping_label_payments_saving_dialog_message">Veuillez patienter...</string>
```

Comment:
Ensure we have at least a string where translators used `...` so that we can check they get substituted to `…` after post-processing.